.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,37 @@
+name: golangci-lint
+on:
+  push:
+    branches:
+      - master
```

Comment:
this will need to get changed to `branches: [ master, development, feat/*, rc/* ]` as fixing a linter issue on a PR will not start the re-check process. Where did you get this file? So we can also apply the patch there.